lib/rubycritic/generators/html/templates/layouts/application.html.erb

```diff
@@ -19,6 +19,13 @@
     <header class="navbar navbar-default navbar-fixed-top">
       <a href="#menu-toggle" class="btn btn-default hidden-lg visible-sm-* hidden-md visible-xs-* pull-left" id="menu-toggle"><i class="fa fa-bars" aria-hidden="true"></i></a>
       <a href="<%= file_path('overview.html') %>"><img src="<%= image_path('logo.png') %>" title="Ruby Critic Logo" width="55"><span class="logo">RUBYCRITIC</span></a>
+      <% if Config.compare_branches_mode? %>
+        <ul class="nav navbar-nav navbar-right">
+          <a href="<%= code_index_path("#{Config.base_root_directory}") %>"><span class="branch"><%= Config.base_branch %></span></a>
+          <a href="<%= code_index_path("#{Config.feature_root_directory}") %>"><span class="branch"><%= Config.feature_branch %></span></a>
+          <a href="<%= code_index_path("#{Config.build_root_directory}") %>"><span class="branch">Build</span></a>
```

Comment:
@Onumis These are the links that appear in header, which will help you navigate and can you be more specific on ``stay on the same page``